src/IndexRange/IndexRange.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <VersionPrefix>1.0.0</VersionPrefix>
-    <TargetFrameworks>net45;net471;netstandard2.0;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks>net45;net47;netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
Why? Cf. bf40be9191567ede232229b1274ed9bc7a005ac3.

Reply:
You're right; this was in .NET 4.7.

Reply:
Ah I did downgrade 471 to 47 that was not my intent.
If anything it should be upgraded to 472

Reply:
No, it should stay at `net47` to provide the greatest compatibility. (The advantage over `net45` is that it won't pull in an unnecessary `System.ValueTuple` dependency.)